include/fiction/utils/layout_utils.hpp

```diff
@@ -600,17 +601,24 @@ CoordinateType random_coordinate(CoordinateType coordinate1, CoordinateType coor
  * @tparam CoordinateType Coordinate Type.
  * @param cell_nw The northwest cell defining the starting point of the area.
  * @param cell_se The southeast cell defining the ending point of the area.
+ * @param obstructed_cell Optional cell which is obstructed and therefore not included in the returned vector.
```

Comment:
This seems oddly specific. Why can there be only a single obstructed cell? Plus, we have a layout type for that: `obstruction_layout`.

Reply:
It is not needed anymore.